components/add_users_to_team_modal/add_users_to_team_modal.tsx

```diff
@@ -26,13 +26,13 @@ type Props = {
     team: Team;
     users: UserProfile[];
     filterExcludeGuests?: boolean;
-    excludeUsers: { [userId: string]: UserProfile };
-    includeUsers: { [userId: string]: UserProfile };
+    excludeUsers: Map<string, UserProfile>;
+    includeUsers: Map<string, UserProfile>;
     onAddCallback: (users: UserProfile[]) => void;
     onHide?: () => void;
 
     actions: {
-        getProfilesNotInTeam: (teamId: string, groupConstrained: boolean, page: number, perPage?: number, options?: Record<string, any>) => Promise<{ data: UserProfile[] }>;
+        getProfilesNotInTeam: (teamId: string, groupConstrained: boolean, page: number, perPage?: number) => Promise<{ data: UserProfile[] }>;
```

Comment:
This is an unrelated change. It is removed since the original function doesn't seem to accept any options.